Sources_v3/Database/DatabaseContainer_Mock.swift

```diff
@@ -60,9 +60,13 @@ extension DatabaseContainer {
     }
     
     /// Synchrnously creates a new ChannelDTO in the DB with the given cid.
-    func createChannel(cid: ChannelId = .unique) throws {
+    func createChannel(cid: ChannelId = .unique, includeMessages: Bool = true) throws {
```

Comment:
`createChannel(includeMessage: true)` removes the channel's messages? A bit confusing, I suggest renaming param to `withMessages` so `if !withMessages { removeAllMessages }`

Reply:
`createChannel(includeMessage: true)` keeps the messages, it's the default. The condition is `if !includeMessages { }`. Maybe you didn't notice the `!`?

Reply:
I see.. `createChannel(includeMessages: true)` as default will create a mock channel **with messages**, and `includeMessages: false` will return a channel without messages. My suggestion still stands, `includeMessages` is a bit confusing, where it's not clear _which_ messages will be included (to me)

Reply:
Renamed to `withMessages` in 5fa87ff